### PR TITLE
feat: add image qr generator

### PR DESCRIPTION
**Pull request by amadeu2**
## Summary
Current behavior:
The QR Code generator creates an SVG wich contains too many nodes to be used inside the Drawer component on mobile. Testers reported that the Drawer opening animation takes up to 5 seconds and hence it tends to an erratic behavoir.

This PR uses a new QR Code generator which uses an image as a css background for just one HTML element, resulting in a smooth Drawer animation.

### Changelog
```
- It was made a clone of the `qrcode-generator` repository and implemented background and foreground options to use in dark mode.
- Implemented a new component to generate a QR Code as image css background.
- Use that component on the Receive drawer.
```

## Type of Change
Please select any type below that is applicable to your changes, and delete those that are not.
- [x] __New__ - a change which implements a new feature

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [ ] MacOS
	- [x] Linux
	- [ ] Windows
- __Mobile__
	- [x] iOS
	- [x] Android

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or modified tests that prove my changes work as intended
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have verified that my latest changes pass CI workflows for testing and linting
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by amadeu2** [comment]
As we have problems scanning inverted QR Codes with the current Capacitor QR scan plugin, we'll keep only the light mode version by now. So I hidded the related code.

File: packages/shared/components/QRImage.svelte [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<script lang="typescript">
+    import qrcode from 'qrcode-generator'
+    import { appSettings } from 'shared/lib/appSettings'
+
+    export let data = ''
+    export let size = 6
+
+    let qr: QRCode
+    let QRTag = ''
+    let background = ''
```
Comment (paul-boegelsack) [inline packages/shared/components/QRImage.svelte:17]: const instead of let?

File: packages/shared/components/QRImage.svelte [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    let foreground = ''
```
Comment (paul-boegelsack) [inline packages/shared/components/QRImage.svelte:18]: const instead of let?

**Review by paul-boegelsack — APPROVED** [review]
Looks good to me. Just two minor things that should be addressed because of warnings.

File: packages/shared/components/QRImage.svelte (hunk 2 above)
Comment (amadeu2) [reply packages/shared/components/QRImage.svelte]: fixed thanks!

**Pull request merged**